src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/SeedCharacteristicsTest.kt

```diff
@@ -0,0 +1,154 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.integration
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.SeedFileType
+
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+class SeedCharacteristicsTest : SeedTestBase() {
+  @Test
+  fun `Attempting to seed characteristic with missing characteristic_name field fails and logs error`() {
+    characteristicRepository.deleteAll() // can't seem to do this in a BeforeEach setUp function
```

Comment:
I deleted all of these and added the following which worked on my machine:

```
@BeforeEach
  fun removeDefaultCharacteristicsFromDatabaseMigrations() {
    characteristicRepository.deleteAll()
  }
```

Reply:
yes, that does work! Thanks. Not sure exactly what was going wrong before...